docker-compose.yml

```diff
@@ -51,8 +51,8 @@ services:
     ports:
       - 9000:9000
     environment:
-      - MINIO_ACCESS_KEY=aws_access_key
-      - MINIO_SECRET_KEY=aws_secret_key
+      - MINIO_ACCESS_KEY=${AWS_ACCESS_KEY_ID}
```

Comment:
I think we should undo this. I intentionally set the access key and secret key to "aws_access_key" and "aws_secret_key", so that the local minio container wouldn't be configured with actual credentials to S3.